infrastructure/time/timestamp.hh

```diff
@@ -1,24 +1,60 @@
 #pragma once
 
+#include <chrono>
+
 #include "infrastructure/comms/serialization/serialization_macros.hh"
 
 namespace jet {
 
 class Timestamp {
  public:
+  // So that everything uses the same block
+  using Clock = std::chrono::system_clock;
```

Comment:
Do you mind using the full name for `std::chrono::system_clock` (or something other than `Clock`) so that I can use the name `Clock` later for a factory class?

Reply:
I've changed the name to `SystemClock`.  Is that acceptable @benkroop ?

Reply:
:+1: